integrations/extensions/starter-kits/google-custom-search/README.md

```diff
@@ -2,26 +2,28 @@
 
 ## Background
 
-This is a starter kit for accessing the Google "Custom Search" JSON API. The "Custom Search" API allows search over a website, collection of websites or the world wide web using the [Google Programmable Search Engine](https://developers.google.com/custom-search/docs/overview) which is a configurable search that allows you to customize search features based on your use-case.
+This is a starter kit for accessing the Google Custom Search JSON API. This API allows search over a website, collection of websites or the world wide web using the [Google Programmable Search Engine](https://developers.google.com/custom-search/docs/overview), which is a configurable search that allows you to customize search features based on your use-case.
 
 The OpenAPI spec in this starter kit includes the following endpoints:
 
 - `GET /customsearch/v1`: Search for content relevant to a given query over the entire web.
 - `GET /customsearch/v1/siterestrict`: Search for content relevant to a given query over a specific collection of websites.
 
 The endpoints are described in detail at:
- 
-1. **Custom Search**: https://developers.google.com/custom-search/v1/reference/rest/v1/cse/list 
+
+1. **Custom Search**: https://developers.google.com/custom-search/v1/reference/rest/v1/cse/list
 2. **Custom Search Site Restricted**: https://developers.google.com/custom-search/v1/reference/rest/v1/cse.siterestrict/list
 
-The Site Restricted API is similar to the Custom Search JSON API except that it has no daily query [limit](https://developers.google.com/custom-search/v1/overview#pricing). It is important to note that the Site Restricted API is used to be run with a Programmable Search Engine which is restricted to only searching specific sites (10 or fewer) whereas the regular Custom Search API can be run with a search engine which searches over the entire web.
+These endpoints have the same arguments and responses.  However, there are two major differences between these endpoints:
+
+- The *Custom Search Site Restricted* endpoint is restricted to searching 10 or fewer websites (each of which can have an unlimited number of pages).  In contrast, *Custom Search* can support any number of websites including all of the web that is indexed by Google.
+- The *Custom Search* endpoint has a [daily query limit](https://developers.google.com/custom-search/v1/overview#pricing), and the *Custom Search Site Restricted* does not.
```

Comment:
This definitely is framed better, thanks! 👍

Reply:
Thanks!